packages/lesswrong/components/sunshineDashboard/ModeratorActions.tsx

```diff
@@ -0,0 +1,428 @@
+import React, { useState } from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import DoneIcon from '@material-ui/icons/Done';
+import SnoozeIcon from '@material-ui/icons/Snooze';
+import AddAlarmIcon from '@material-ui/icons/AddAlarm';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import RemoveCircleOutlineIcon from '@material-ui/icons/RemoveCircleOutline';
+import VisibilityOutlinedIcon from '@material-ui/icons/VisibilityOutlined';
+import OutlinedFlagIcon from '@material-ui/icons/OutlinedFlag';
+import classNames from 'classnames';
+import { useUpdate } from '../../lib/crud/withUpdate';
+import { useCreate } from '../../lib/crud/withCreate';
+import moment from 'moment';
+import { LOW_AVERAGE_KARMA_COMMENT_ALERT, LOW_AVERAGE_KARMA_POST_ALERT, MODERATOR_ACTION_TYPES, RATE_LIMIT_ONE_PER_DAY } from '../../lib/collections/moderatorActions/schema';
+import FlagIcon from '@material-ui/icons/Flag';
+import Input from '@material-ui/core/Input';
+import { isLowAverageKarmaContent } from '../../lib/collections/moderatorActions/helpers';
+import { sortBy } from 'underscore';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  row: {
+    display: "flex",
+    alignItems: "center",
+    marginTop: 8,
+    marginBottom: 8
+  },
+  modButton:{
+    marginTop: 6,
+    marginRight: 16,
+    cursor: "pointer",
+    '&:hover': {
+      opacity: .5
+    }
+  },
+  snooze10: {
+    color: theme.palette.primary.main,
+    fontSize: 34,
+    marginTop: 4
+  },
+  permissionsButton: {
+    padding: 6,
+    paddingTop: 3,
+    paddingBottom: 3,
+    border: theme.palette.border.normal,
+    borderRadius: 2,
+    cursor: "pointer",
+    whiteSpace: "nowrap",
+    marginRight: 8
+  },
+  permissionDisabled: {
+    border: "none"
+  },
+  notes: {
+    border: theme.palette.border.faint,
+    borderRadius: 2,
+    paddingLeft: 8,
+    paddingRight: 8,
+    paddingTop: 4,
+    paddingBottom: 4,
+    marginTop: 8,
+    marginBottom: 8,
+  },
+});
+
+interface UserContentCountPartial {
+  postCount?: number,
+  commentCount?: number
+}
+
+export function getCurrentContentCount(user: UserContentCountPartial) {
+  const postCount = user.postCount ?? 0
+  const commentCount = user.commentCount ?? 0
+  return postCount + commentCount
+}
+
+export function getNewSnoozeUntilContentCount(user: UserContentCountPartial, contentCount: number) {
+  return getCurrentContentCount(user) + contentCount
+}
```

Comment:
I think these also exist in `SunshineNewUsersInfo`

Reply:
Thanks. Removed from SunshineNewUsersInfo.